generators/docker-compose/templates/_docker-compose.yml

```diff
@@ -61,10 +61,12 @@ services:
         extends:
             file: jhipster-console.yml
             service: jhipster-console
+    <%_ if (composeApplicationType === 'microservice') { _%>
```

Comment:
I'm not sure about this `composeApplicationType`. Where is it defined ? Shouldn't it be microservice or gateway ?
[EDIT] Never mind it is in prompt.js. No problem.